contentcuration/contentcuration/static/js/edit_channel/preview/views.js

```diff
@@ -82,44 +82,42 @@ var PreviewView = BaseViews.BaseView.extend({
     generate_preview:function(force_load){
         if(this.current_preview){
             extension = this.current_preview.file_format;
-
-                var preview_template;
-                switch (extension){
-                    case "png":
-                    case "jpg":
-                    case "jpeg":
-                        preview_template = require("./hbtemplates/preview_templates/image.handlebars");
-                        break;
-                    case "pdf":
-                    case "PDF":
-                    case "vtt":
-                    case "srt":
-                        preview_template = require("./hbtemplates/preview_templates/document.handlebars");
-                        break;
-                    case "mp3":
-                        preview_template = require("./hbtemplates/preview_templates/audio.handlebars");
-                        break;
-                    case "mp4":
-                        preview_template = require("./hbtemplates/preview_templates/video.handlebars");
-                        break;
-                    case "perseus":
-                        preview_template = require("./hbtemplates/preview_templates/exercise.handlebars");
-                        break;
-                    case "zip":
-                        preview_template = require("./hbtemplates/preview_templates/html5.handlebars");
-                        break;
-                    default:
-                        preview_template = require("./hbtemplates/preview_templates/default.handlebars");
-                }
-                this.$("#preview_window").html(preview_template({
-                    source: this.current_preview.storage_url,
-                    extension:this.current_preview.mimetype,
-                    checksum:this.current_preview.checksum,
-                    subtitles : this.get_subtitles()
-                }));
-                if(force_load && this.current_preview.recommended_kind === "video"){
-                    $("#preview_window video").load();
-                }
+            var preview_template;
+            switch (extension){
+                case "png":
```

Comment:
Does having all of these cases benefit us in any way? Do we _not_ want them to hit the default value?